haas/api.py

```diff
@@ -333,10 +319,10 @@ def node_detach_network(node, nic, network):
 
     Raises BadArgumentError if the network is not attached to the nic.
     """
-    db = model.Session()
-    node = _must_find(db, model.Node, node)
-    network = _must_find(db, model.Network, network)
-    nic = _must_find_n(db, node, model.Nic, nic)
+    db = local.db
```

Comment:
(After refreshing my memory): I left it in here in a few places as a shorthand, where it makes sense and where it doesn't is a bit subjective.